symbolicatorprocessor/symbolicator.go

```
@@ -81,12 +89,13 @@ func (ns *basicSymbolicator) limitedSymbolicate(ctx context.Context, line, colum
 	}()
 
 	smc, ok := ns.cache.Get(url)
+	ns.telemetryBuilder.ProcessorSourceMapCacheSize.Record(ctx, int64(ns.cache.Len()), ns.attributes)
 
 	if !ok {
 		source, sMap, err := ns.store.GetSourceMap(ctx, url)
 
 		if err != nil {
-			return nil, err
```

Should we also be returning the error here after the fetch failure? This might make things fail silently.

---

I think I completely removed this by accident 😬 thank you for catching it